Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml

```diff
@@ -350,125 +349,153 @@
-                                    </MenuItem.Icon>
-                                </MenuItem>
-                                <MenuItem Command="{Binding RunSelectedGroupCommand}" 
-                                      Header="{Resx ResxName=Rubberduck.Resources.UnitTesting.TestExplorer, Key=TestExplorer_ContextMenuRunGroup}">
-                                    <MenuItem.Icon>
-                                        <Image Source="{StaticResource RunSelectedGroupImage}" />
-                                    </MenuItem.Icon>
-                                </MenuItem>
-                                <MenuItem Command="{Binding RunAllTestsCommand}" 
-                                      Header="{Resx ResxName=Rubberduck.Resources.UnitTesting.TestExplorer, Key=TestExplorer_ContextMenuRunAll}">
-                                    <MenuItem.Icon>
-                                        <Image Source="{StaticResource RunAllTestsImage}" />
-                                    </MenuItem.Icon>
-                                </MenuItem>
-                                <MenuItem Command="{Binding IgnoreTestCommand}"
-                                      Header="{Resx ResxName=Rubberduck.Resources.UnitTesting.TestExplorer, Key=TestExplorer_TestToggle_Ignore}"
-                                      Visibility="{Binding DisplayIgnoreTestLabel, Converter={StaticResource BoolToVisibility}}">
-                                    <MenuItem.Icon>
-                                        <Image Source="{StaticResource IgnoreTestImage}" />
-                                    </MenuItem.Icon>
-                                </MenuItem>
-                                <MenuItem Command="{Binding UnignoreTestCommand}"
-                                      Header="{Resx ResxName=Rubberduck.Resources.UnitTesting.TestExplorer, Key=TestExplorer_TestToggle_Unignore}"
-                                      Visibility="{Binding DisplayUnignoreTestLabel, Converter={StaticResource BoolToVisibility}}">
-                                    <MenuItem.Icon>
-                                        <Image Source="{StaticResource UnignoreTestImage}" />
-                                    </MenuItem.Icon>
-                                </MenuItem>
-                            </ContextMenu>
-                        </DataGrid.ContextMenu>
-                        <i:Interaction.Behaviors>
-                            <local:TestExplorerRowMouseOverBehavior MouseOverTest="{Binding MouseOverTest, Mode=TwoWay}" MouseOverGroup="{Binding MouseOverGroup, Mode=TwoWay}" />
-                            <controls:GroupItemExpandedBehavior ExpandedState="{Binding ExpandedState, Mode=TwoWay}" />
-                        </i:Interaction.Behaviors>
-                    </controls:GroupingGrid>
-                </ScrollViewer>
-            </Grid>
+				<ScrollViewer VerticalScrollBarVisibility="Disabled" HorizontalScrollBarVisibility="Auto">
```

Comment:
Again, a tabs vs. spaces issue. Could you please take a look at your settings under XAML?

Reply:
XAML settings have insert spaces selected. I've no clue why it's adding tabs.